### PR TITLE
doc: improve submodule update command - README.md

### PR DESCRIPTION
**Pull request by zdover23**
Improve the portion of README.md that directs the reader to update the submodules. Previously, the text assumed that the reader knew that they should cd into the ceph directory. Now the text is explicit about the necessity of this action.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,9 +34,9 @@ command on a system that has git installed:
 	git clone https://github.com/ceph/ceph.git
 
 When the ceph/ceph repository has been cloned to your system, run the following
-command to check out the git submodules associated with the ceph/ceph
-repository: 
+commands to move into the cloned ceph/ceph repository and to check out the git submodules associated with it :
```
Comment (anthonyeleven) [inline README.md:37]: I would backtick `ceph/ceph`

**Review by anthonyeleven — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (zdover23) [reply README.md:37]: I was on the fence, but now that I have your vote I am not on the fence.

**Comment by zdover23** [comment]
jenkins test make check

**Comment by zdover23** [comment]
jenkins test make check

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/53000 - Reef backport
https://github.com/ceph/ceph/pull/53001 - Quincy backport

**Comment by anthonyeleven** [comment]
M04R 6R33|\| T1L3z!

**Pull request merged**